app/institutions/dashboard/-components/institutional-users-list/styles.scss

```diff
@@ -96,21 +90,122 @@
     margin-left: 4px;
 }
 
-th .sort-arrow {
+.sort-arrow {
     display: inline-block;
     vertical-align: middle;
     color: #fff;
 }
 
-.table-filters {
+.select-container {
+    width: 100%;
+    display: flex;
+    justify-content: flex-end;
+    float: right;
+}
+
+.select {
+    min-width: 120px;
+    padding: 7px 16px 7px 14px;
+    border-color: #ddd;
+    border-radius: 2px;
+    color: #337ab7;
+    text-align: center;
+    margin: 15px;
+
+    span {
+        margin-left: 0;
+    }
+}
+
+.filter-container {
     display: flex;
     justify-content: flex-end;
     align-items: center;
     width: 100%;
-    padding: 10px 0;
+}
+
+.dropdown-trigger {
+    background-color: #fff;
+    border: 1px solid #ddd;
+    color: #007bff;
+    border-radius: 4px;
+    padding: 9px;
+    font-size: 14px;
+    cursor: pointer;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    box-shadow: 0 1px 2px rgba(0, 0, 0, 0.05);
+}
+
+.dropdown-trigger:hover {
+    background-color: #f8f9fa;
+    border-color: #007bff;
+}
```

Comment:
If you change the button that this style is used for to `<Button>`, you should get most of this styling for free. I think the only thing this class would need to have is likely `color: $color-text-blue-dark` and `padding: 9px`